content/docs/intro/concepts/_index.md

```diff
@@ -9,49 +9,292 @@ menu:
 aliases: ["/docs/reference/concepts/"]
 ---
 
-Pulumi is a modern infrastructure as code platform. It includes a CLI, runtime, libraries, and a hosted service that, working together, deliver a robust way of provisioning, updating, and managing cloud infrastructure. Instead of YAML or a domain-specific language (DSL), Pulumi leverages existing, familiar programming languages, including TypeScript, JavaScript, Python, Go, and .NET, and their native tools, libraries, and package managers.
+Pulumi is a modern infrastructure as code platform. It includes a CLI, runtime, libraries, and a hosted service working together to deliver a robust way of provisioning, updating, and managing cloud infrastructure.
 
-This section will teach you the core concepts behind Pulumi, including how to author your Pulumi programs, the CLI you will use to deploy those programs, how projects (and their related concept, "stacks") help you to organize your environments, and how Pulumi works under the hood.
+Instead of YAML or a domain-specific language (DSL), Pulumi leverages existing programming languages---TypeScript, JavaScript, Python, Go, and .NET---and their native tools, libraries, and package managers to interact with cloud resources through the Pulumi SDK.
 
 > If this is your first time using Pulumi, you likely want to begin with [the Getting Started guide]({{< relref "/docs/get-started" >}}) for your cloud of choice. It will walk you through an [AWS]({{< relref "/docs/get-started/aws" >}}), [Azure]({{< relref "/docs/get-started/azure" >}}), [GCP]({{< relref "/docs/get-started/gcp" >}}), or [Kubernetes]({{< relref "/docs/get-started/kubernetes" >}}) deployment from start to finish.
 
-The following topics describe the core concepts behind Pulumi in more detail:
+## Programming Model Overview {#programmingmodel}
```

Comment:
In the way we are now structuring this content - I think this no longer really is the "programming model".

```suggestion
## Pulumi Overview {#overview}
```